lib/virtualGraphs.js

```diff
@@ -9,7 +9,7 @@ const list = conn => {
   }).then(httpBody);
 };
 
-const add = (conn, name, mappings, options) => {
+const add = (conn, name, mappings, options, params) => {
   const headers = conn.headers();
```

Comment:
A question would be whether to pass in 'db' through params, or whether it should be passed in by itself.

Reply:
So, my thoughts here are these:

- "params" is something we use typically (if not exclusively) for query parameters. Since this isn't a query parameter, I don't think it should be on "params".
- However, I also don't think it should be on "db" (or equivalent) by itself, because that isn't very forward-thinking (we might want to add additional options in the future, and we don't want them to all be named arguments).
- So, how about an argument not named "params" that is still an object on which various bits of metadata can be passed in? I'd support calling it "meta" or "metadata".